pkg/kube/cluster-init.sh

```diff
@@ -300,20 +300,30 @@ check_start_containerd() {
                 logmsg "Started k3s-containerd at pid:$containerd_pid"
         fi
         if [ -f /etc/external-boot-image.tar ]; then
+                # Is containerd up?
+                if ! /var/lib/k3s/bin/k3s ctr -a /run/containerd-user/containerd.sock info > /dev/null 2>&1; then
+                        logmsg "k3s-containerd not yet running for image import"
+                        return
```

Comment:
Is `check_start_containerd` in a loop? It didn't look like it in the rest of the script (but I easily could have missed it). If not, then this will return without doing anything and not retry.

Reply:
check_start_containerd is called in the main while loop every 15 seconds

Reply:
Ah, ok, I missed that part. In that case, this works fine.